themes/classic/assets/css/custom.css

```diff
@@ -2,3 +2,22 @@
  * Custom code goes here.
  * A template should always ship with an empty custom.css
  */
+@media screen and (max-width: 767px) {
+	.category-cover{
+		display: none !important;
```

Comment:
Why every style has `!important` declaration? 
If the style will affect a component globally, the style should be directly modified at the component level.
If the style is a custom one, it is better to create a specific class for that, so the style could be reused in the future (if needed).

Reply:
You are right, I will fix this.